### PR TITLE
adding privacy policy

### PR DESCRIPTION
**Pull request by tiffanymarchi**
Added privacy policy to site (as a hidden page not listed on the nav menu) and added hyperlink to site footer.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-        <div class="newsletter-title">Stay connected</div>
-        <p>Join us today to be the first to know about our latest product launches.</p>
+        <div class="newsletter-title">Stay Connected</div>
+        <p>Join us today and be the first to know about our latest product launches.</p>
         <form id="newsletter-form" class="newsletter-form">
             <input required name="email" type="email" class="gv-input" placeholder="Email">
             <button type="submit" class="gv-button">Submit</button>
         </form>
-        <span class="message">We won’t send you spam, unsubscribe at any time</span>
+        <span class="message">We won’t send you spam (unsubscribe at any time). View our Privacy Policy <a href="https://docs.gravitee.io/apim/3.x/gravitee_privacy_policy.html">here</a>.</span>
```
Comment (gaetanmaisse) [inline _includes/footer.html:9]: I think this page shouldn't be under `/apim/3.x`, something like `https://docs.gravitee.io/gravitee_privacy_policy.html`

File: pages/privacypolicy/site-privacy-policy.adoc [hunk 2]
```diff
@@ -0,0 +1,354 @@
+:page-sidebar: apim_3_x_sidebar
```
Comment (gaetanmaisse) [inline pages/privacypolicy/site-privacy-policy.adoc:1]: To remove?

File: pages/privacypolicy/site-privacy-policy.adoc [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+:page-permalink: apim/3.x/gravitee_privacy_policy.html
```
Comment (gaetanmaisse) [inline pages/privacypolicy/site-privacy-policy.adoc:2]: Just `gravitee_privacy_policy.html`?

File: pages/privacypolicy/site-privacy-policy.adoc [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+:page-folder: apim/installation-guide/amazon
```
Comment (gaetanmaisse) [inline pages/privacypolicy/site-privacy-policy.adoc:3]: To remove

File: pages/privacypolicy/site-privacy-policy.adoc [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+:page-liquid:
+:page-layout: apim3x
```
Comment (gaetanmaisse) [inline pages/privacypolicy/site-privacy-policy.adoc:5]: To remove

File: pages/privacypolicy/site-privacy-policy.adoc [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+:page-description: Gravitee.io API Management - Installation Guide - Amazon - API Management
```
Comment (gaetanmaisse) [inline pages/privacypolicy/site-privacy-policy.adoc:6]: To update

File: pages/privacypolicy/site-privacy-policy.adoc [hunk 7]
```diff
@@ -0,0 +1,354 @@
...
+:page-keywords: Gravitee.io, API Platform, API Management, API Gateway, oauth2, openid, documentation, GDPR, privacy policy, api
+:page-toc: false
+
+:gravitee-package-name: graviteeio-apim-3x
```
Comment (gaetanmaisse) [inline pages/privacypolicy/site-privacy-policy.adoc:10]: To remove

**Review by gaetanmaisse — CHANGES_REQUESTED** [review]

File: _includes/footer.html (hunk 1 above)
Comment (tiffanymarchi) [reply _includes/footer.html]: fixed! thank you for this suggestion.

**Comment by tiffanymarchi** [comment]
Made Gaetan's suggested edits (all the ones that were applicable), corrected broken link (404 error) in the process.

**Pull request merged**